packages/hub/src/hub.ts

```diff
@@ -439,6 +439,16 @@ export class Hub implements HubInterface {
     return session;
   }
 
+  /**
+   * Returns if default PII should be sent to Sentry and propagated in ourgoing requests
+   * when Tracing is used.
+   */
+  public shouldSendDefaultPii(): boolean {
+    const { client } = this.getStackTop();
+    const options = client && client.getOptions();
+    return Boolean(options && options.sendDefaultPii);
+  }
```

Comment:
For future readers and reviewers: This is similar to how the [Python SDK](https://github.com/getsentry/sentry-python/blob/85208da360e3ab6fa4e38b202376353438e4f904/sentry_sdk/hub.py#L75-L80) is doing it